Identity_Infrastructure/Repositories/GenericRepository.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Identity_Application.Interfaces.Repository;
```

Comment:
if we don't have errors handling on this lvl, we need to do it  on lvl of query and command(where necessary)

Reply:
added error handling